pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "hive-agent"
+version = "0.0.1"
+description = "Client library to interact with Hive Agents"
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.9"
```

Comment:
should be at least 3.11, is your current python version 9 and it currently works for you?